src/middlewares/getUserFromResourceServer.js

```diff
@@ -0,0 +1,16 @@
+
+const getUser = (accessToken, refreshToken, profile, done) => {
+  console.log(`user : ${JSON.stringify(profile)}`);
+  const method = profile.provider;
+  const { id } = profile;
+  const firstName = profile.name.firstName || profile.name.givenName;
+  const lastName = profile.name.lastName || profile.name.familyName;
+  const { gender } = profile;
+  const email = (profile.emails) ? profile.emails[0].value : null;
+  const user = {
+    id, method, firstName, lastName, email, gender,
+  };
+  done(null, user);
+};
+
+export default getUser;
```

Comment:
This file doesnt seem to be tested! Run `npm run coverage` and look at the coverage report.

Reply:
I faced difficulties testing it but now is fixed. 
Thanks @MuhanguziDavid.